include/git2/status.h

```diff
@@ -298,31 +298,6 @@ GIT_EXTERN(int) git_status_file(
 	git_repository *repo,
 	const char *path);
 
-/**
```

Comment:
Please squash this commit into the initial commit where you created that function